components/table/Filters/IFieldFilterType.cs

```diff
@@ -2,16 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
 using System.Linq.Expressions;
-using System.Text;
+using Microsoft.AspNetCore.Components;
 
-namespace AntDesign.FilterExpression
+namespace AntDesign.Filters
 {
-    public interface IFilterExpression
+    public interface IFieldFilterType
     {
-        TableFilterCompareOperator GetDefaultCompareOperator();
+        TableFilterCompareOperator DefaultCompareOperator { get; }
+        RenderFragment<TableFilterInputRenderOptions> FilterInput { get; }
+        bool SupportsCompareOperator(TableFilterCompareOperator compareOperator);
```

Comment:
I reviewed the IFieldFilterType code and found that the SupportsCompareOperator method only determines which operators to display, but not in which order. I think we can use a list instead, so that the user can easily specify the type and order. As long as we loop the render in the order of the list.

```
    StringFieldFilterType stringFieldFilterType=new();

    protected override void OnInitialized()
    {
        stringFieldFilterType.SupportsCompareOperators.AddRange(new[]
        {
            TableFilterCompareOperator.LessThan,
        });

        base.OnInitialized();
    }

```